src/Picasso_LevelSet.hpp

```diff
@@ -63,19 +63,21 @@ class LevelSet
         _dx = _mesh->localGrid()->globalGrid().globalMesh().cellSize( 0 );
 
         // Extract parameters.
-        const auto& params = ptree.get_child( "level_set" );
+        const auto& params = inputs["level_set"];
 
         // Get the Hopf-Lax redistancing parameters.
-        _redistance_secant_tol =
-            params.get<double>( "redistance_secant_tol", 0.25 );
-        _redistance_max_secant_iter =
-            params.get<int>( "redistance_max_secant_iter", 10 );
-        _redistance_num_random_guess =
-            params.get<int>( "redistance_num_random_guess", 5 );
-        _redistance_projection_tol =
-            params.get<double>( "redistance_projection_tol", 1.0e-4 );
-        _redistance_max_projection_iter =
-            params.get<int>( "redistance_max_projection_iter", 200 );
+        if ( params.count( "redistance_secant_tol" ) )
```

Comment:
Why use `count()` for these instead of the more appropriate `contains()` method?

Reply:
I think I was a bit unfamiliar with the api at that point.. fixed all instances